arviz/plots/autocorrplot.py

```diff
@@ -56,13 +57,10 @@ def autocorrplot(posterior, var_names=None, max_lag=100, symmetric_plot=False, c
     for (var_name, selection, x), ax in zip(plotters, axes.flatten()):
         if combined:
             x = x.flatten()
-        y = x - x.mean()
-        y = np.correlate(y, y, mode=2)
-        y = y / np.abs(y).max()
-        midpoint = len(y) // 2
+        y = autocorr(x)
         ax.vlines(x=np.arange(min_lag, max_lag),
-                  ymin=np.zeros(max_lag - min_lag),
```

Comment:
this was allowing for `symmetric_plot=True`, though I am not sure I see the use of that option any more.

Reply:
Specifically, I think this is fine, but either here or in a followup PR we should remove the `symmetric_plot` option.

Reply:
I removed the option. `autocorr(x)` already removes symmetry, so we would have to reintroduce it after that computation (by essentially appending a reversed copy of `y` to itself). I think this is a rare, specialized usecase that I have not seen used in actual plots too much.